### PR TITLE
Refactors butcherable

### PR DESCRIPTION
**Pull request by CheffieGithub**
## About the PR 
Its not a boolean but its used as one, now it has defines:
`BUTCHER_NOT_ALLOWED`
`BUTCHER_ALLOWED`
`BUTCHER_YOU_MONSTER`

Adjusted some butcherable values where it makes sense:
- Can't butcher robotic
- `BUTCHER_YOU_MONSTER` for seals and walruses
- Allows butchering of more small animals

Fixes some stuff in the owlery file while I was there it was annoying.

## Why's this needed? 
Clarity of what butcherable is.

### CODE REVIEW

**Comment by DimWhat** [comment]
I suggest adding bees to the "BUTCHER_YOU_MONSTER" section

File: code/WorkInProgress/Hydrothings.dm [hunk 1]
```diff
@@ -1,12 +1,8 @@
 //Content related to Operation Fuckable Owls
```
Comment (TheTyrant25) [inline code/WorkInProgress/Hydrothings.dm:1]: seems like most of these refactors here are unrelated to butchering

**Review by TheTyrant25 — COMMENTED** [review]
Seems mostly good except for these owlrey changes which seem to have bled over from another project. The actual find replace of butcherable looks to have worked fine

File: code/WorkInProgress/Hydrothings.dm (hunk 1 above)
Comment (CheffieGithub) [reply code/WorkInProgress/Hydrothings.dm:1]: `Fixes some stuff in the owlery file while I was there it was annoying.` Its just removing whitespace and adding missing slashes

Comment (DisturbHerb) [reply code/WorkInProgress/Hydrothings.dm:1]: i would atomize those changes out regardless

Comment (CheffieGithub) [reply code/WorkInProgress/Hydrothings.dm:1]: seems like too small of a change to me to warrant its own PR

Comment (DisturbHerb) [reply code/WorkInProgress/Hydrothings.dm:1]: no change is ever too small :beesleep:

**Review by frawhst — APPROVED** [review]
lgtm. Whitespace stuff is ok even if it produces a fat diff.

**Comment by CheffieGithub** [comment]
will fix the conflict once I have some working cleared
